app/controllers/defence_requests_controller.rb

```diff
@@ -10,7 +10,7 @@ class DefenceRequestsController < BaseController
   helper_method :defence_request_path_with_tab
 
   def show
-    @tab = params[:tab]
+    @tab = params[:tab] && %(interview case-details).include?(params[:tab]) ?  params[:tab] : nil
```

Comment:
The tab javascript we are using has an `activatefirst` data attribute which we are defaulting to when to tab has been specified.